packages/theme-predictive-search-component/README.md

```diff
@@ -0,0 +1,57 @@
+## Description
+
+A component that controls the state of a predictive search UI.
+
+## Example
+
+```js
+const predictivesearch = new PredictiveSearch({
+  selectors: {
+    input: '[data-predictive-search-input="header"]',
+    reset: '[data-predictive-search-reset="header"]',
```

Comment:
The logic for a reset button is not completely fleshed out atm.

Reply:
I will remove this line.

Reply:
Can we delete line 11 then?